### PR TITLE
Add search by zip

### PR DESCRIPTION
**Pull request by goodevilgenius**
OpenWeatherMap allows getting current weather [based on zip code](https://openweathermap.org/current#zip). This commit allows this library to implement this type of search by adding a potential query in the form of "zip:<zipcode>", e.g., "zip:77070" would search in Houston, TX.

### CODE REVIEW

File: Cmfcmf/OpenWeatherMap.php [hunk 1]
```diff
@@ -179,10 +179,13 @@ public function getApiKey()
...
      * @return CurrentWeather The weather object.
      *
-     * There are three ways to specify the place to get weather information for:
+     * There are four ways to specify the place to get weather information for:
      * - Use the city name: $query must be a string containing the city name.
      * - Use the city id: $query must be an integer containing the city id.
      * - Use the coordinates: $query must be an associative array containing the 'lat' and 'lon' values.
+     * - Use the zip code: $query must be a string, prefixed with "zip:"
+     *
+     * Zip code may specifify country. e.g., "zip:77070" (Houston, TX, US) or "zip:500001,IN" (Hyderabad, India)
```
Comment (cmfcmf) [inline Cmfcmf/OpenWeatherMap.php]: Typo: specify

File: Cmfcmf/OpenWeatherMap.php [hunk 2]
```diff
@@ -670,6 +673,9 @@ private function buildQueryUrlParameter($query)
                 return 'id='.implode(',', $query);
             case is_numeric($query):
                 return "id=$query";
+            case is_string($query) && strpos($query, 'zip:') === 0:
+                $sub_query = str_replace('zip:', '', $query);
```
Comment (cmfcmf) [inline Cmfcmf/OpenWeatherMap.php]: Please use CamelCase: `$subQuery`

**Comment by goodevilgenius** [comment]
@cmfcmf Changes made.

**Comment by cmfcmf** [comment]
Thank you!

**Pull request merged**